CRM/Report/Form/Mailing/Summary.php

```diff
@@ -149,11 +149,13 @@ public function __construct() {
       'order_bys' => array(
         'start_date' => array(
           'title' => ts('Start Date'),
+          'dbAlias' => 'MAX(mailing_job_civireport.start_date)'
```

Comment:
I _think_ this should be MIN. We want the earliest start date, and latest end date.